test/widgets/autocomplete_test.dart

```diff
@@ -108,37 +108,39 @@ void main() {
   group('ComposeAutocomplete', () {
 
     Finder findNetworkImage(String url) {
+      String trimmedUrl  = url.substring(0, url.length -4); //Remove the extension (.png) from the URL
```

Comment:
Let's do this with a [`RegExp`](https://api.flutter.dev/flutter/dart-core/RegExp-class.html) instead of just chopping characters off the end, which seems more confusing than necessary if `url` ever ends up ending in something other than ".png".